UnityProject/Assets/Scripts/Clothing/RaceSOSingleton.cs

```diff
@@ -22,4 +22,14 @@ public static bool TryGetRaceByName(string raceName, out PlayerHealthData race)
 		race = null;
 		return false;
 	}
+
+	public static List<PlayerHealthData> GetPlayerRaces()
+	{
+		var list = new List<PlayerHealthData>();
```

Comment:
embrace LINQ

```suggestion
		return Instance.Races.Where(race => race.Base.CanBePlayerChosen).ToList();
```

as a second suggestion, I would replace any future mention of "race" with "specie" since it is more semantically correct.

Reply:
done 👍